content/concepts/loaders.md

```diff
@@ -32,6 +33,64 @@ Loaders are [resolved similar to modules](/concepts/module-resolution/). A loade
 
 ### Referencing Loaders
 
-By convention, loaders are usually named as `XXX-loader`, where `XXX` is the context name. For example, `json-loader`.
+By convention, loaders are usually named as `xxx-loader`, where `xxx` is the context name. For example, `babel-loader`.
 
+In the following example, we will configure webpack to run all files which end with `.js` through the `babel-loader`.
+
+All webpack loaders are standard npm modules and installed via npm from command line as follows:
+
+
+## Usage
+
+To set the `loader` property, you define the `module.rules` array value in your webpack config:
+
+**webpack.config.js**
+
+```javascript
+const config = {
+  ...
+  module: {
+    rules: [
+      {test: /\.js$/, loader: 'babel-loader', exclude: /node_modules/}, // transform js files
+      {test: /\s[a|c]ss$/, loader: 'sass-loader'}                       // transform sass files
```

Comment:
Why before the other styling loader? Shouldn't sass bits get evaluated first (evaluation order for loaders is from bottom to top if I remember right).